doc/user/requirements.txt

```diff
@@ -1,4 +1,4 @@
 mkdocs==1.6.1
-mkdocs-material==9.5.49
-mike@git+https://github.com/jimporter/mike.git@f0522f245e64687dd18384fbd86b721175711474
```

Comment:
Why were we using this specific version?

Reply:
Because the sorting of `dev-2.x` to the top was not supported before. Now it is.